### ISSUE TITLE
[BUG] Document not uploaded due to UnsupportedImageTypeError

### ISSUE THREAD

**Issue by Northguy** [paperless-ngx/paperless-ngx#675]
### Description

I downloaded a random washing machine manual and tried to upload it into the https://demo.paperless-ngx.com/ environment. Document is readable in Adobe Acrobat, but is not consumed by paperless-nxt. Paperless-nxt throws UnsupportedImageTypeError

### Expected behavior

Document uploaded into Paperless-nxt

### Steps to reproduce

Open demo environment
Login with demo credentials
Upload attached document
Wait for error

[Samsung B1013J Washing Machine.pdf](https://github.com/paperless-ngx/paperless-ngx/files/8450454/Samsung.B1013J.Washing.Machine.pdf)

### Webserver logs

```bash
[2022-04-08 09:08:24,974] [ERROR] [paperless.consumer] Error while consuming document Samsung B1013J Washing Machine.pdf: UnsupportedImageTypeError: ICC profile corrupt or not readable

Traceback (most recent call last):

  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 479, in icc

    self._icc = ImageCms.ImageCmsProfile(iccbytesio)

  File "/usr/local/lib/python3.9/site-packages/PIL/ImageCms.py", line 172, in __init__

    self._set(core.profile_frombytes(profile.read()))

OSError: cannot open profile from string

The above exception was the direct cause of the following exception:

Traceback (most recent call last):

  File "/usr/src/paperless/src/paperless_tesseract/parsers.py", line 249, in parse

    ocrmypdf.ocr(**args)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/api.py", line 337, in ocr

    return run_pipeline(options=options, plugin_manager=plugin_manager, api=True)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/_sync.py", line 370, in run_pipeline

    pdfinfo = get_pdfinfo(

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/_pipeline.py", line 156, in get_pdfinfo

    return PdfInfo(

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 904, in __init__

    self._pages = _pdf_pageinfo_concurrent(

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 681, in _pdf_pageinfo_concurrent

    executor(

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/_concurrent.py", line 82, in __call__

    self._execute(

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/builtin_plugins/concurrency.py", line 135, in _execute

    result = future.result()

  File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 439, in result

    return self.__get_result()

  File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result

    raise self._exception

  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run

    result = self.fn(*self.args, **self.kwargs)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 638, in _pdf_pageinfo_sync

    page = PageInfo(pdf, pageno, infile, check_pages, detailed_analysis)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 716, in __init__

    self._gather_pageinfo(pdf, pageno, infile, check_pages, detailed_analysis)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 762, in _gather_pageinfo

    for ci in _process_content_streams(

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 565, in _process_content_streams

    yield from _find_regular_images(container, contentsinfo)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 483, in _find_regular_images

    yield ImageInfo(name=draw.name, pdfimage=pdfimage, shorthand=draw.shorthand)

  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/pdfinfo/info.py", line 353, in __init__

    pim_icc = pim.icc

  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 483, in icc

    raise UnsupportedImageTypeError(

pikepdf.models.image.UnsupportedImageTypeError: ICC profile corrupt or not readable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "/usr/src/paperless/src/documents/consumer.py", line 245, in try_consume_file

    document_parser.parse(self.path, mime_type, self.filename)

  File "/usr/src/paperless/src/paperless_tesseract/parsers.py", line 301, in parse

    raise ParseError(f"{e.__class__.__name__}: {str(e)}")

documents.parsers.ParseError: UnsupportedImageTypeError: ICC profile corrupt or not readable

[2022-04-08 09:08:27,764] [INFO] [paperless.consumer] Consuming 11003870-000-03_11003870-03.pdf

[2022-04-08 09:08:27,780] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2022-04-08 09:08:27,876] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2022-04-08 09:08:27,955] [DEBUG] [paperless.consumer] Parsing 11003870-000-03_11003870-03.pdf...

[2022-04-08 09:08:27,972] [DEBUG] [paperless.parsing.tesseract] Extracted text from PDF file /tmp/paperless/paperless-upload-55pz7t1q

[2022-04-08 09:08:29,175] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': '/tmp/paperless/paperless-upload-55pz7t1q', 'output_file': '/tmp/paperless/paperless-kad5qof2/archive.pdf', 'use_threads': True, 'jobs': 1, 'language': 'eng', 'output_type': 'pdfa', 'progress_bar': False, 'skip_text': True, 'clean': True, 'deskew': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': '/tmp/paperless/paperless-kad5qof2/sidecar.txt'}

[2022-04-08 09:09:19,762] [DEBUG] [paperless.parsing.tesseract] Incomplete sidecar file: discarding.
```

### Screenshots

![image](https://user-images.githubusercontent.com/1189058/162406028-16b0f25a-4f7e-49f1-b39a-1a89cd25b906.png)

### Paperless-ngx version

1.6.0

### Host OS

Demo environment

### Installation method

Other (please describe above)

### Browser

Chrome

### Configuration changes

n/a

### Other

n/a

**Comment by shamoon** [comment]
Well isn’t this just reporting that the pdf file is messed up? `ICC profile corrupt or not readable`, not sure if that’s really a bug. If you generate / use a clean pdf I assume it would work fine as it usually does

**Comment by Northguy** [comment]
Hi,

It was one of three files I tested to see if I would like to step into using Paperless. I did not scan/create the file myself, just used a commercially available manual from the Samsung website. I have attached it to the bug report, should someone want to test for themselves.

As the file is readable without issues when opened in Acrobat, but crashes when importing in paperless, this would indicate for me that the importing mechanism/error catching is less robust than regular Acrobat. I do not feel that authoring pdfs should be necessary, just for the sake of archiving purposes if they are readable without issues by acrobat reader.

If the dev. Team thinks there are more pressing issues to pursue or have a lack of capacity to investigate, feel free to close this report.

**Comment by qcasey** [comment]
@paperless-ngx/backend correct me if I'm wrong, but this is an issue with just OCR?

> I do not feel that authoring pdfs should be necessary, just for the sake of archiving purposes if they are readable without issues by acrobat reader.

I agree with this completely. There shouldn't be an error severe enough to interrupt the upload. I'll play around with this later today.

**Comment by stumpylog** [comment]
> correct me if I'm wrong, but this is an issue with just OCR?

That's what it looks like to me.  Either OCRmyPDF or pikepdf.  I can find a few references to this error in their issues, but nothing that appears to be a definite fix.  The file loads fine in Sumatra as well.

**Comment by apollo13** [comment]
While OCR errors shouldn't interrupt the process, it would also help to report those issues to pikepdf. See my example here https://github.com/pikepdf/pikepdf/issues/346

The problem here seems to be that OCR needs to extract the image info and pikepdf fails to read the ICC. This might as well be a bug in pillow, but let's see…

**Comment by oscherler** [comment]
OCRmyPDF was just tagged at version 13.4.6, which includes PR https://github.com/ocrmypdf/OCRmyPDF/pull/972 that logs a warning instead of failing when this happens.

**Comment by apollo13** [comment]
Will be "fixed" by #1105 most likely

**Comment by stumpylog** [comment]
ocrmypdf was upgraded in #1116, which converts this from an error to a warning, so files will be consumable, but could be missing content (if any was in the particular failing image).
